index.html

```diff
@@ -48,78 +57,127 @@ <h2>Tonic</h2>
-                    <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+            <div class="deskImg3"></div>
+            <div class="grid-colum">
+                <h4>Tonic</h4>
+                <h4 class="deskTitle1">Facebook 360</h4>
+                <ul class="mobile">
+                    <li class="company" id="deskHideCanopy">CANOPY</li>
+                    <li class="deskCompany2">FACEBOOK</li>
+                    <li id="deskHideJob">Back End Dev</li>
+                    <li class="mobileHideJob">Full Stack Dev</li>
+                    <li>2013</li>
                 </ul>
+                <p class="mobileP">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <p class="deskP">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
+                <div class="mobileTeck">
+                    <ul class="teck-list">
+                        <li><button class="mobile-btn" type="submit">HTML</button></li>
+                        <li><button class="mobile-btn" type="submit">CSS</button></li>
+                        <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+                    </ul>
+                </div>
+                <div class="deskTeck">
+                    <ul class="deskTeck-list">
+                        <li><button class="desk-btn" type="submit">HTML</button></li>
+                        <li><button class="desk-btn" type="submit">Ruby on Rails</button></li>
+                        <li><button class="desk-btn" type="submit">CSS</button></li>
+                        <li><button class="desk-btn" type="submit">JavaScript</button></li>
+                    </ul>
+                </div>
+                <button class="see-mobile-btn" type="submit">See Projects</button>
             </div>
-            <button class="see-mobile-btn" type="submit">See Projects</button>
         </section>
         <!-- card4 -->
         <section class="card4">
             <div class="mobileimage4"></div>
-            <h3>Multi-Post <br> Stories</h3>
-            <ul class="mobile">
-                <li class="company">CANOPY</li>
-                <li>Back End Dev</li>
-                <li>2013</li>
-            </ul>
-            <p class="mobile-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <div class="teck">
-                <ul class="teck-list">
-                    <li><button class="mobile-btn" type="submit">HTML</button></li>
-                    <li><button class="mobile-btn" type="submit">CSS</button></li>
-                    <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+            <div class="deskImg4"></div>
+            <div class="grid-colum">
+                <h3>Multi-Post <br> Stories</h3>
+                <h3 class="deskTitle1">Uber Navigation</h3>
```

Comment:
- [ ] kindly fix the card button so it follows the figma design. yours is longer 
- [ ] please the text on the card is not bold. kindly remove the font weight given to it.
- [ ] add a little margin left to the card content so it aligns with the original design. 
yours
![Screenshot (1240)_LI](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/65298048/4bd09453-403b-474e-8afa-5d0c1c28af19)
figma design
![Screenshot (1240)](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/65298048/543b4a44-3cb7-46d4-953c-ded3f9179052)